src/main/java/de/zalando/ep/zalenium/proxy/DockerSeleniumRemoteProxy.java

```diff
@@ -231,6 +231,7 @@ private TestSession createNewSession(Map<String, Object> requestedCapability) {
             boolean videoRecording = Boolean.parseBoolean(getCapability(requestedCapability, ZaleniumCapabilityType.RECORD_VIDEO, "true"));
             setVideoRecordingEnabledSession(videoRecording);
         }
+        String testFileNameTemplate = getCapability(requestedCapability, ZaleniumCapabilityType.TEST_FILE_NAME_TEMPLATE, "");
```

Comment:
Could you please add to the docs an example of how this capability should look like? Also an example of how the file name would be based on the capability. It could go here https://github.com/zalando/zalenium/blob/master/docs/_posts/2000-01-07-usage.md#test-configuration-options